packages/flutter_tools/lib/src/base/net.dart

```diff
@@ -12,31 +13,46 @@ const int kNetworkProblemExitCode = 50;
 
 /// Download a file from the given URL and return the bytes.
 Future<List<int>> fetchUrl(Uri url) async {
```

Comment:
We may want to add a timeout to the invocation of this in `cache.dart`

Reply:
Why? Without the assets, we can't do anything useful (modulo https://github.com/flutter/flutter/issues/6491).

Reply:
Just to cause the tool to fail as opposed to retrying ad infinitum